src/Kestrel.Core/Internal/HttpConnectionBuilderExtensions.cs

```diff
@@ -11,12 +11,22 @@ public static class HttpConnectionBuilderExtensions
     {
         public static IConnectionBuilder UseHttpServer<TContext>(this IConnectionBuilder builder, ServiceContext serviceContext, IHttpApplication<TContext> application)
         {
-            return builder.UseHttpServer(Array.Empty<IConnectionAdapter>(), serviceContext, application);
+            return builder.UseHttpServer(serviceContext, application, protocols: HttpProtocols.Http1);
```

Comment:
why the named parameters here?

Reply:
Talked to @davidfowl. I'm removing those extension methods and leaving only the new ones (with the new `HttpProtocols` param).